feature/Makefile

```diff
@@ -6,6 +6,7 @@ SRCDIR  = $(TOP)/src
 OBJDIR  = $(TOP)/obj
 TOOLDIR = $(TOP)/tool
 LIBVMAF = $(TOP)/../wrapper/libvmaf.a
+LIBVMAF_HEADER = $(TOP)/../wrapper/src
```

Comment:
This seems to introduce circular dependency between /wrapper and /feature. Previously it was only /wrapper depends on /feature. Can we avoid this?

Reply:
True, I have had this in mind. The reason this happens is because read_frame (in feature/common) and feature extractors, need some stuff from libvmaf.h. Perhaps there is a better way to do this.

Reply:
In this case, move the dependent stuff from libvmaf.h to feature/common to avoid the circular depedency?

Reply:
Adding a todo about that inside.